src/typescript/azure-openapi-validator/rules/IntegerTypeMustHaveFormat.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from "../rule"
+export const IntegerTypeMustHaveFormat: string = "IntegerTypeMustHaveFormat"
+
+rules.push({
+  id: "R4013",
+  name: IntegerTypeMustHaveFormat,
+  severity: "error",
+  category: "ARMViolation",
+  mergeState: MergeStates.individual,
+  openapiType: OpenApiTypes.arm | OpenApiTypes.dataplane,
+  appliesTo_JsonQuery: "$..*[?(@.type == 'integer')]",
+  *run(doc, node, path) {
+    const msg: string = `The integer type does not have a format, please add it.`
+    const formats = ["int32", "int64"]
+    if (!node.format || formats.indexOf(node.format) === -1) {
```

Comment:
please add integer type "int32" or "int64"?

Reply:
not this, To add is  `"fotmat":"int32" `